includes/rest/class-outbox-controller.php

```diff
@@ -198,7 +210,7 @@ public function get_items( $request ) {
 			$response['orderedItems'][] = $this->prepare_item_for_response( $outbox_item, $request );
 		}
 
-		$response = $this->prepare_collection_response( $response, $request );
+		$response = $this->prepare_collection_response( $response, $request, $outbox_query->found_posts );
```

Comment:
Instead of passing `$outbox_query->found_posts` to `prepare_collection_response()`, couldn't we just keep it as the default? `'totalItems'` should be the correct amount before the response gets passed to `prepare_collection_response()`.